k8s/helmfile/env/common/ui.values.yaml.gotmpl

```diff
@@ -0,0 +1,24 @@
+resources:
```

Comment:
This is fine right now, but I think we might want to discuss as a team "what goes where" and whether we even want to specify resources in common.

Reply:
I agree. We could aim to create a new [ADR](/doc/adr) after that discussion